ivy/functional/frontends/jax/numpy/fft.py

```diff
@@ -11,6 +11,15 @@ def fft(a, n=None, axis=-1, norm=None):
     return ivy.fft(a, axis, norm=norm, n=n)
 
 
+@to_ivy_arrays_and_back
+def fft2(a, s=None, axes=(-2, -1), norm=None):
+    if norm is None:
+        norm = "backward"
+    if ivy.dtype(a) == "complex128":
```

Comment:
I think you can omit the check for dtype as function will only recieve complex types. you can directly do this `return ivy.array(ivy.fft2(a, s=s, dim=axes, norm=norm), dtype=ivy.dtype(a))`

Reply:
Yes seems right to just make it inclusive of these dtypes, Pushed changes all seems to be working well!